app/models/note.rb

```diff
@@ -2,6 +2,8 @@ class Note < ActiveRecord::Base
   belongs_to :candidate
   belongs_to :user
   before_save :update_candidate
+  validates :comment, presence: true, length: { in: 10..2000 }
```

Comment:
@pascallaliberte @cyprusad I rebased with @miranmirza's new master added a nice note for users to understand when things didn't save correctly.
![screen shot 2015-06-11 at 3 21 42 pm](https://cloud.githubusercontent.com/assets/8007635/8116027/d15830d6-103c-11e5-98dd-9b372f836dc5.png)